### ISSUE TITLE
sound effect editor accessibility: volume and frequency sliders are not tabbable

### ISSUE THREAD

**Issue by eanders-ms** [microsoft/pxt-microbit#4642]
Changing sound volume or frequency on PC requires a mouse.

Expected: Slider controls should be tab-indexed, and movable up and down with arrow keys.

Browser: Firefox
OS: Windows 11

**Comment by microbit-matt-hillsdon** [comment]
micro:bit intend to look at this as part of the keyboard nav work.
